core/types/src/attributes.rs

```diff
@@ -53,7 +53,11 @@ impl Display for Attributes {
                     write!(f, "Flags: {}", flags)?
                 }
             }
-            None => write!(f, "Flags: (none)")?,
+
```

Comment:
When using the masking of all the bits `0xFFFF_FFFF_FFFF_FFFF` it seemed wrong to be outputting `(none)` here, figured if any error happens a display of the hex value would be better